docs/data-point-definition.md

```diff
@@ -0,0 +1,32 @@
+# Data Point Definition
+Gro defines a "data point" as a discrete result produced by our API. When using the `get_data_points()` function, you are returned an array of points, each of which is a Python dictionary object that looks something like:
+```py
+{
+  u'input_unit_scale': 1,
+  u'region_id': 1038,
+  u'end_date': u'2000-12-31T00:00:00.000Z',
+  u'input_unit_id': 17,

```

Comment:
Let's remove `'input_unit_scale'` and change `'input_unit_id'` to `'unit_id'` like we did in the descriptions below so there's not a mis-match.

Also, let's re-order these and the descriptions below so they're presented more like order of importance. Something like:

1. start_date
2. end_date
3. value
4. unit_id
5. reporting_date
6. metric_id
7. item_id
8. region_id
9. frequency_id
10. source_id
11. partner_region_id

Rationale:
1. start/end/value/unit is the primary info being conveyed
2. reporting date is sometimes part of that primary info
3. metric/item/region/frequency/source is secondary just to double-check what data you got in response since you generally assume it's the same as what you requested
4. partner's only sometimes relevant to that secondary category

Reply:
I've left the order in the sample response, since that's the order that is returned in actual responses. But I've adjusted the more detailed definitions to the order you suggested.

I've also updated to a new sample response, since the previous one has been migrated.

Reply:
The order in the actual response has no significance and could change whenever. No guarantees made about that. So I don't think there's anything wrong with re-ordering them in our sample for readability's sake